### DISCUSSION TITLE
ROF and Ammo Expenditure

### DISCUSSION THREAD

**Discussion by rlewisfr** [fvtt-fria-ligan/twilight2000-foundry-vtt#81] [Q&A]
Great work on this one, what a fantastic way to get back into T2K for the first time since the 80's!

Had a question about the ammunition expenditure calculation on the attack.  Not sure what the relationship between the ROF and the ammo expended is.  When I select 0 ROF, I get 1 shot (expected).  But selected anything higher than 0 leads to some inconsistent results.  For example, on an attack with an M4A1 (ROF of 6) these are my results:

ROF | Ammo Dice (Hit) | Ammo Expenditure
-- | -- | --
0 | 0 | 1
1 | 1 | 4
2 | 2 | 8
3 | 3 | 9
4 | 4 | 18
5 | 5 | 20
6 | 4 | 19
6 | 5 | 19
6 | 6 | 27
6 | 5 | 16
6 | 6 | 22
6 | 5 | 16
6 | 5 | 24
6 | 6 | 22

The reason I went with Ammo Dice that actually hit is that I was thinking there might be a relationship there, but it is unclear.  The average is about 3.92 bullets/ammo die hitting, or 3.62 bullets/ammo die regardless of hit.

Wondering what I can tell the players about ammo used in proportion to the ROF slider they would like to select.

Thank you,

**Comment by tinwe** [comment]
The amount of ammo spent doesn't depend on how many successes you get, but rather the values on the ammo dice rolled and that is what seems to be missing from your table above.
https://pasteboard.co/M7gwvE3Akv37.jpg
